task/backend/analytical_storage.go

```diff
@@ -153,25 +153,24 @@ func (as *AnalyticalStorage) FindRuns(ctx context.Context, filter influxdb.RunFi
 		filterPart = fmt.Sprintf(`|> filter(fn: (r) => r.runID > %q)`, filter.After.String())
 	}
 
-	parsedAfterTime := time.Time{}
-	parsedBeforeTime := time.Now()
 	constructedTimeFilter := ""
 	if feature.TimeFilterFlags().Enabled(ctx) {
```

Comment:
> Should I reopen the mirrored branch on idpe and make the same changes and set you as a reviewer?

Completely up to you as many of these were small nits rather than functionality changes. that reminds me though, feature flags are a cloud concept and can be removed from oss (is my understanding, and should be verified)